### ISSUE TITLE
TextInput looses references

### ISSUE THREAD

**Issue by plotic** [heap/react-native-heap#226]
Prerequisites:
react-native 0.62.2
React Navigation 5
heap library version 0.13.0
Integrate the library using  `heapAutoInit` true

I declare a textInput ref, and i have a button that calls the focus function at that time the textInput ref is null. 

```
const textInput = useRef(null)
*******
const enableInputHandler = () => {
     textInput.current.focus() // <---- current IS NULL
}
*******
<TouchableOpacity
                activeOpacity={1}
                onPress={enableInputHandler}
                style={{
                     styles.profileInputContainer
}}>
*******
<TextInput
                        ref={textInput}
                        style={{
                               styles.input                          
                        }}
                        autoCapitalize='none'
                        autoCorrect={false}
                        keyboardType={keyboardType ? keyboardType : 'default'}
                        value={formatValue(inputValue)}
                        onChangeText={inputValueHandler}
                        onEndEditing={saveNewInput}
                        onFocus={onFocus}
/>                        

```

**Comment by jmtaber129** [comment]
Hi @plotic, this looks like an issue that should be fixed as of 0.14.0 (released a few days ago).  Please upgrade to this newer version, and let us know if this issue persists.

**Comment by plotic** [comment]
Hey @jmtaber129 thanks , i'll check with the latest version

**Comment by bnickel** [comment]
Closing because it was resolved in 0.14.0.
